install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -90,14 +90,27 @@ resources:
           kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value core/account)
 
           wget https://github.com/istio/istio/releases/download/{{ properties['installIstioRelease'] }}/istio-{{ properties['installIstioRelease'] }}-linux.tar.gz
+
           tar xzf istio-{{ properties['installIstioRelease'] }}-linux.tar.gz
 
-          wget -P /root/helm/ https://storage.googleapis.com/kubernetes-helm/helm-v2.9.1-linux-amd64.tar.gz
-          tar xf /root/helm/helm-v2.9.1-linux-amd64.tar.gz  -C /root/helm/
+          wget -P /root/helm/ https://storage.googleapis.com/kubernetes-helm/helm-v2.11.0-linux-amd64.tar.gz
+          tar xf /root/helm/helm-v2.11.0-linux-amd64.tar.gz  -C /root/helm/
 
           export PATH="$PATH:/root/istio-{{ properties['installIstioRelease'] }}/bin::/root/helm/linux-amd64/"
           cd /root/istio-{{ properties['installIstioRelease'] }}
 
+          kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml
+
+          kubectl apply -f install/kubernetes/helm/subcharts/certmanager/templates/crds.yaml
+
+          sleep 5
```

Comment:
is there a way to make this event driven (e.g,. poll something until the new changes show up)?  what happens if 5 seconds isn't long enough?

Reply:
removed;not necessary